docs/eventing/sources/_index.md

```diff
@@ -34,14 +34,14 @@ kn source list
 
 | Name | API Version | Maintainer | Description |
 | -- | -- | -- | -- |
-| [APIServerSource](https://github.com/knative/eventing/blob/master/pkg/apis/sources/v1/apiserver_types.go) | v1 | Knative  | Brings Kubernetes API server events into Knative. The APIServerSource fires a new event each time a Kubernetes resource is created, updated or deleted. |
-| [AWS SQS](https://github.com/knative-sandbox/eventing-awssqs/blob/master/pkg/apis/sources/v1alpha1/aws_sqs_types.go)  | v1alpha1 | Knative | Brings [AWS Simple Queue Service](https://aws.amazon.com/sqs/) messages into Knative. The AwsSqsSource fires a new event each time an event is published on an [AWS SQS topic](https://aws.amazon.com/sqs/).  |
-| [Apache Camel](https://github.com/knative-sandbox/eventing-camel/blob/master/pkg/apis/sources/v1alpha1/camelsource_types.go) | v1alpha1   | Knative    | Enables use of [Apache Camel](https://github.com/apache/camel) components for pushing events into Knative. A CamelSource is an event source that can represent any existing [Apache Camel component](https://github.com/apache/camel/tree/master/components), that provides a consumer side, and enables publishing events to an addressable endpoint. Each Camel endpoint has the form of a URI where the scheme is the ID of the component to use. CamelSource requires [Camel-K](https://github.com/apache/camel-k#installation) to be installed into the current namespace. See the [CamelSource](https://github.com/knative-sandbox/eventing-camel/tree/master/samples) example. |
+| [APIServerSource](https://github.com/knative/docs/blob/master/docs/eventing/sources/apiserversource.md) | v1 | Knative  | Brings Kubernetes API server events into Knative. The APIServerSource fires a new event each time a Kubernetes resource is created, updated or deleted. |
```

Comment:
Since this is part of the `knative/docs` repository we should use a relative import path.

Reply:
Like in this file https://raw.githubusercontent.com/knative/docs/main/docs/eventing/broker/alternate/README.md we use `[link](../kafka-broker.md)`

Reply:
Correct the path as suggested.